resources/ext/ezbake.conf

```diff
@@ -34,7 +34,7 @@ ezbake: {
                ]
              }
 
-      debian: { dependencies: ["puppet-agent"],
+      debian: { dependencies: ["puppet-agent < 1.3.0"],
```

Comment:
The deprecated forms < and > were confusingly used to mean earlier/later or equal, rather than strictly earlier/later, and must not appear in new packages (though dpkg still supports them with a warning).

I think you want some `<<` here instead.